meta.json

```diff
@@ -21,5 +21,10 @@
       "model": "erh:viamrtsp:rtsp-mjpeg"
     }
   ],
-  "entrypoint": "bin/viamrtsp"
-}
\ No newline at end of file
+  "build": {
+    "build": "bash build.sh",
+    "path": "module.tar.gz",
+    "arch" : ["linux/amd64", "linux/arm64"]
+  },
+  "entrypoint": "viamrtsp"
```

Comment:
I recommend not changing the entry point as eliot's fix to the module deploy problem didn't change the entrypoint: https://github.com/erh/viamrtsp/commit/4d7999f204a58b6564cfe2c0386e330fbf6c6f5c